### PR TITLE
095 docs

### LINKED ISSUE #395: [Concept] Finalization
The next catapult-server (core engine) version will add a finalization layer. The main docs will require an update in order to reflect the new feature.

### LINKED ISSUE #399: Document VRF & Voting new transactions
Next release will come with two new transaction types. Additional info will be disclosed with the finalization design.

Catbuffer changes: https://github.com/nemtech/catbuffer/compare/783fdaf732db81d247fe862a8703e3b619aadf57...master

### PR DESCRIPTION
**Pull request by dgarcia360**
Closes #399

Ongoing work, please do not merge.

### CODE REVIEW

File: source/concepts/harvesting.rst [hunk 1]
```diff
@@ -56,24 +70,27 @@ Local harvesting is secure as long as no one accesses your node instance, which
 Delegated harvesting
 ********************
 
-An eligible account may also delegate its importance score to a :ref:`remote node <delegated-harvesting>` for harvesting.
+Delegated harvesting allows using the importance score of an account to create new blocks and receive rewards without having to run a node locally.
 
-Delegated harvesting enables an account to use a **proxy private key** that can be shared with a node securely.
-In other words, you can use the importance score of your account to create new blocks without running a node.
+An eligible account can delegate its importance score to a brand new **proxy account**
+The private key of the proxy account can be shared securely with any node that is running local harvesting thatdelegated harvesting connections.
```
Comment (matthewkim93) [inline source/concepts/harvesting.rst:76]: missed a space between "thatdelegated"

File: source/concepts/harvesting.rst [hunk 2]
```diff
@@ -56,24 +70,27 @@ Local harvesting is secure as long as no one accesses your node instance, which
...
 
-To enable delegated harvesting, the account owner has to link its **importance score** to a remote account announcing an :ref:`AccountLinkTransaction <account-link-transaction>`.
+To enable delegated harvesting, the account owner has to link its importance score to the proxy account by announcing an :ref:`AccountKeyLinkTransaction <account-key-link-transaction>`.
+In order for the remote account to be accepted for delegated harvesting, the remote account should not have sent nor received any transaction or be linked to another account previously.
+
+.. note:: The remote account cannot be involved in any transaction while it is a delegated account.
 
 Then, the account needs to send a `special encrypted message <https://github.com/nemtech/NIP/blob/master/NIPs/nip-0009.md>`_ to the node via a :doc:`TransferTransaction <transfer-transaction>`.
-The message must contain the remote's account **proxy private key**  encrypted using AES, so that only the recipient will be able to decipher it.
+The message must contain the remote's account **proxy private key**  encrypted using AES, making the transaction only readable by the recipient.
```
Comment (matthewkim93) [inline source/concepts/harvesting.rst:90]: The message must contain the remote's account **proxy private key**  encrypted using AES, making the transaction readable only by the recipient.
